### PR TITLE
Pokemon Transfer fix

### PR DESCRIPTION
**Pull request by ghost**
Please test this. I have but want feedback.

### CODE REVIEW

**Comment by LNSDISCORDER** [comment]
um, when you say you tested. Were you awake at the time? Jokes aside, can you confirm that this branch builds for testing? https://github.com/Afraidium/PokeMobBot/tree/development

**Comment by ghost** [comment]
Yea, Built it, ran it, didn't have a million pokes in inventory afterwards
@LNSDISCORDER What is the issue?

**Comment by DeadElle** [comment]
Needs to be cleaned and reworked, I'm actually working at the same thing, PM me on Discord if you want ;)

File: PoGo.PokeMobBot.Logic/PoGo.PokeMobBot.Logic.csproj [hunk 1]
```diff
@@ -30,10 +30,21 @@
...
+    <Reference Include="Google.Protobuf, Version=3.0.0.0, Culture=neutral, PublicKeyToken=a7d26565bac4d604, processorArchitecture=MSIL">
+      <HintPath>..\packages\Google.Protobuf.3.0.0-beta4\lib\net45\Google.Protobuf.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <HintPath>..\packages\Newtonsoft.Json.9.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="POGOProtos">
+      <HintPath>..\..\..\..\..\..\..\NecroBot-master\packages\POGOProtos.1.4.0\lib\net45\POGOProtos.dll</HintPath>
```
Comment (ActuallyTrent) [inline PoGo.PokeMobBot.Logic/PoGo.PokeMobBot.Logic.csproj:46]: We will not accept copy + pasted code from NecroBot.

**Comment by ActuallyTrent** [comment]
Line 46 of the Logic.csproj - You have references to NecroBot. Please do not copy and paste code from their project trying to bring it here.

**Pull request closed without merging**